### ISSUE TITLE
Fix warning with Jinja2 template delimiters in when statement

### ISSUE THREAD

**Issue by hectcastro** [azavea/ansible-java#24]
Unclear what version of Ansible emitted this (wasn't my machine). Probably something that needs to be fixed though so that we don't accidentally evaluate the condition incorrectly:

```
TASK [azavea.java : Add OpenJDK PPA] *******************************************
Tuesday 01 August 2017  16:39:17 -0400 (0:00:00.746)       0:00:28.420 ********
 [WARNING]: when statements should not include jinja2 templating delimiters
such as {{ }} or {% %}. Found: {{ ansible_distribution_version |
version_compare('14.04', '=') and java_major_version | version_compare('8',
'>=') }}
```

**Comment by tnation14** [comment]
Closed via #25

**Comment by tnation14** [comment]
Closed via #25
